src/app/homebrew-sharing/tome-share/tome-share.component.html

```diff
@@ -5,7 +5,7 @@
     </span>
     <span class="flex-spacer"></span>
     <span>
-      By {{owner.username}}
+      By {{owner?.username || 'Loading...'}}
```

Comment:
So this will say: "By Loading..."?

Maybe you should `{owner?.username ? 'By ${owner?.username}' : 'Loading'}` (or whatever)?

Reply:
Or better, a spinner 😃